### ISSUE TITLE
Add an option for keep Menu Bar visible while "Themed Title Bar" is enable in Dark Mode

### ISSUE THREAD

**Issue by Ahsan400** [sublimehq/sublime_merge#1074]
**Problem description**

In windows if we enable dark mode and also enable "Themed Title Bar" it hides "Menu Bar". If "Themed Title Bar" that solves one problem that is menu bar become visible but create another which is Menu bar & Title bar become white.

**Preferred solution**

Add an option in settings that will keep Menu bar visible even after eneble  "Themed Title Bar" like the first screenshot.

**Screenshots**

![IMG_20210226_151930](https://user-images.githubusercontent.com/58716264/109287750-34969c80-784e-11eb-93b1-def051392526.png)

**Comment by scotte** [comment]
Yes, please - the collapsed menu bar is hard to use, and it's not obvious on how to fix it. Thanks.
